docs/installation/installing-with-npm.md

```diff
@@ -254,7 +254,7 @@ To use different asset paths, also complete the below step(s).
   ``` SCSS
   // Include images from /images/govuk-frontend and fonts from /fonts
   $govuk-images-path: “/images/govuk-frontend”;
```

Comment:
The same is also true for the images path.

Reply:
true, updated 👍